estuary_updater/handlers/errata.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from estuary_updater.handlers.base import BaseHandler
+from estuary.models.errata import Advisory
+from estuary.models.bugzilla import BugzillaBug
+from estuary.utils.general import timestamp_to_datetime
+
+import requests
+import requests_kerberos
+
+
+class ErrataHandler(BaseHandler):
+    """A handler for dist-git related messages."""
+
+    @staticmethod
+    def can_handle(msg):
+        """
+        Determine if this handler can handle this message.
+
+        :param dict msg: a message to be analyzed
+        :return: a bool based on if the handler can handle this kind of message
+        :rtype: bool
+        """
+        return msg['topic'] == '/topic/VirtualTopic.eng.errata.activity.status'
+
+    def handle(self, msg):
+        """
+        Handle an Errata tool message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        advisory_id = msg['body']['headers']['errata_id']
+
+        url = self.config['estuary_updater.errata_api_url'] + 'erratum/{}'.format(advisory_id)
+        response = requests.get(url, auth=requests_kerberos.HTTPKerberosAuth())
+        advisory_json = response.json()
+
+        advisory_type = msg['body']['headers']['type'].lower()
+        # TODO: The way DateTime objects are handled will need to be changed once we stop using the
+        # API, as the strings are formatted differently in the message bus messages
+        actual_ship_date = advisory_json['errata'][advisory_type]['actual_ship_date']
```

Comment:
Optional: You should consider making `advisory_json['errata'][advisory_type]` a variable since it's used quite often.